### ISSUE TITLE
How would a decoder be implemented?

### ISSUE THREAD

**Issue by alphamupsiomega** [Cadene/skip-thoughts.torch#2]
The original decoder from Kiros is here: https://github.com/ryankiros/skip-thoughts/tree/master/decoding

How would this be implemented in this repo?

**Comment by Cadene** [comment]
You can reproduce the Kiros' setup. You would need to implement the decoder which takes as input the outputs of the encoder (it is not too hard using the code in this repo as a starting point) and as output a sequential list (one step per word) of logits (proba of words). Then the dataset. I advise you to look at different implementations online.
